CRM/Activity/BAO/Query.php

```diff
@@ -631,27 +629,6 @@ public static function selectorReturnProperties() {
     return $properties;
   }
 
-  /**
-   * Custom form rules.
-   *
-   * @param array $fields
-   * @param array $files
-   * @param CRM_Core_Form $form
-   *
-   * @return bool|array
-   */
-  public static function formRule($fields, $files, $form) {
```

Comment:
We can re-add something generic for these if we think it's important (I'm inclined to put a bit on this)